### PR TITLE
Add emax64-pdumper 20180619 (27.0.50)

### PR DESCRIPTION
**Pull request by Retia-Adolf**

### CODE REVIEW

File: bucket/emax64-pdumper.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+    "homepage": "https://ntemacs.sourceforge.io",
+    "description": "\r\n* 64-bit Windows build of emacs-pdumper branch (27.0.50).\r\n* Pathed with ImageMagick 7 support. Binaries and libs inclued.\r\n* PDF-Tools (epdfinfo.exe) inclued. [emax.7z packge]\r\n* Features[1]: \"XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS MODULES THREADS JSON CANNOT_DUMP LCMS2\"\r\n* Optimized clean build. Options[2]: \"--without-compress-install --without-dbus --with-modules 'CFLAGS= -O2 -g3'\"\r\n* Built and tested on a clean Windows 10 system.\r\n* addpm modified to not create/update any registry HOME entries. It only creates shortcuts as it should.",
```
Comment (r15ch13) [inline bucket/emax64-pdumper.json]:
```suggestion
    "description": "64-bit Emacs for Windows",
```

File: bucket/emax64-pdumper.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            "EMAX64"
+        ],
+        [
+            "bin\\emacsclientw.exe",
+            "EMAX64 Client",
+            " -c -n -a \"\""
+        ]
+    ],
+    "checkver": {
+        "url": "https://github.com/m-parashar/emax64/releases.atom",
```
Comment (r15ch13) [inline bucket/emax64-pdumper.json]:
```suggestion
        "url": "https://github.com/m-parashar/emax64/releases",
```

**Comment by r15ch13** [comment]
Could be added to extras bucket.

**Comment by Retia-Adolf** [comment]
lukesampson/scoop-extras#1916

**Pull request closed without merging**